spec/controllers/container_image_controller_spec.rb

```diff
@@ -31,6 +31,26 @@
                                           :url  => "/container_image/show/#{container_image.id}"}])
   end
 
+  describe "#show" do
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+      @user = FactoryGirl.create(:user)
```

Comment:
I donder if we need the `@user` here at all, cannot we just pass that to the `login_as` call?